rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/BuildConfigurationEndpoint.java

```diff
@@ -380,4 +378,16 @@ BuildConfiguration restoreRevision(
     @Path("/supported-parameters")
     Set<org.jboss.pnc.dto.response.Parameter> getSupportedParameters();
 
+    @Operation(summary = "Provides list of default alignment parameters according to build type.",
+            description = "Provides default parameters for build configuration set in resource config file.",
```

Comment:
"set in resource config file" - it is implementation detail, which shouldn't be on the REST API. The important information is, that these strings are the default params.